website/versioned_docs/version-28.x/TestingAsyncCode.md

```diff
@@ -115,7 +115,11 @@ If `done()` is never called, the test will fail (with timeout error), which is w
 
 If the `expect` statement fails, it throws an error and `done()` is not called. If we want to see in the test log why it failed, we have to wrap `expect` in a `try` block and pass the error in the `catch` block to `done`. Otherwise, we end up with an opaque timeout error that doesn't show what value was received by `expect(data)`.
 
-_Note: `done()` should not be mixed with Promises as this tends to lead to memory leaks in your tests._
+:::caution
+
+`done()` should not be mixed with promises as this tends to lead to memory leaks in your tests.
```

Comment:
This is a runtime error now I think. Might make sense to mention? #10512

Reply:
Good idea. I double checked, runtime error is there since v27. So the note is tweak from v27 and up.